lib/oban/crontab/cron.ex

```diff
@@ -40,6 +40,53 @@ defmodule Oban.Crontab.Cron do
   defp matches_rule?(:weekdays, weekday, datetime), do: weekday == Date.day_of_week(datetime)
 
   @spec parse!(input :: binary()) :: t()
+  def parse!("@annually"), do: parse!("@yearly")
+
+  def parse!("@yearly"),
```

Comment:
That was my first approach, but figured it wasn't work the roundtrip through the parser since the return value will always be the same.

It would be much more readable if the formatter didn't put the structs on multiple lines.

I'd be happy to run some benchmarks if you want.

Reply:
Ah, cool. I prefer the terseness as well, will make the change.